data/tutorials/lg_04_labels.md

```diff
@@ -505,16 +514,16 @@ main ()
   close (fd);                                            // line 15
 }
 ```
-When I compile the code I get a whole bunch of errors including:
+When I compile the code, I get a whole bunch of errors including:
```

Comment:
Not directly a comment on this PR, for future ref. It was "we" in the beginning of the page and now switched to "I", maybe we should open another PR to fix these inconsistencies.

Reply:
I think you're right. Consistency in the voice is desirable. I opened https://github.com/ocaml/ocaml.org/issues/1154.

Reply:
Consistency is key!

Reply:
I went in and removed the other first person singular instances in this document.